src/components/avatar/_avatar.scss

```diff
@@ -32,27 +32,18 @@ $includeHtml: false !default;
       width: 100%;
 
       &--icon {
-        padding-top: 3px;
         background-color: $avatarDefaultBackgroundColor;
       }
     }
 
     &--small {
       width: $avatarSmallSize;
       height: $avatarSmallSize;
-
-      .sg-avatar__image--icon {
```

Comment:
do we use `sg-avatar__image--icon` for anything else? If not, we should get rid of it from the HTML IMO